components/service/glean/src/main/java/mozilla/components/service/glean/EventMetricType.kt

```diff
@@ -102,6 +102,13 @@ data class EventMetricType(
             eventKeys
         }
 
+        if (Glean.configuration.logEvents) {
```

Comment:
Let's leave this out from this PR for now. Let's consider doing this as part of future improvement, when needed (do you mind filing a bug?).

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1518505